### PR TITLE
Megafauna Boss Music

### PR DESCRIPTION
**Pull request by monster860**
#### Read our contributing.md if you haven't already: https://github.com/yogstation13/Yogstation/blob/master/.github/CONTRIBUTING.md

# General Documentation

### Intent of your Pull Request

Adds music when you're fighting lavaland megafauna. Music is made by me.
https://cdn.discordapp.com/attachments/134722036353204224/812253348484218901/2021-02-19_04-22-36.mp4

here's the music links (you'll have to do right click -> save link as if you're in chrome):
[legion](https://github.com/monster860/Yogstation-TG/raw/b3e2c9de470744b6c7a8903ffae8186dfcc6b33f/sound/lavaland/music/pants.ogg)
[ash drake](https://github.com/monster860/Yogstation-TG/raw/b3e2c9de470744b6c7a8903ffae8186dfcc6b33f/sound/lavaland/music/adder.ogg)
[bubblegum](https://github.com/monster860/Yogstation-TG/raw/b3e2c9de470744b6c7a8903ffae8186dfcc6b33f/sound/lavaland/music/blobpaste.ogg)
[colossus](https://github.com/monster860/Yogstation-TG/raw/b3e2c9de470744b6c7a8903ffae8186dfcc6b33f/sound/lavaland/music/communism.ogg)
[hierophant](https://github.com/monster860/Yogstation-TG/raw/b3e2c9de470744b6c7a8903ffae8186dfcc6b33f/sound/lavaland/music/headpats.ogg)
[king goat](https://github.com/monster860/Yogstation-TG/raw/b3e2c9de470744b6c7a8903ffae8186dfcc6b33f/sound/lavaland/music/kilogram.ogg)
[king goat, second part](https://github.com/monster860/Yogstation-TG/raw/b3e2c9de470744b6c7a8903ffae8186dfcc6b33f/sound/lavaland/music/kilogram2.ogg)

### Why is this change good for the game?

music good

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
it adds music

### What should players be aware of when it comes to the changes your PR is implementing?
it adds music

### What general grouping does this PR fall under? 
For example, Engineering changes, Medical rebalancing, TEG tweaks, etc.?
megafauna

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
no

# Changelog

:cl:  
soundadd: Adds lavaland boss music
/:cl:

### CODE REVIEW

**Comment by athena413** [comment]
bubblegum's and ash drake's theme sound annoyingly similar, but that's a bit nitpicky

other than that i have a couple questions:

how does it determine when to play the music? does it just play for the entire z level once the fight starts?

and: if you run away from the megafauna will the music stop playing?

**Comment by ChesterTheCheesy** [comment]
as long as it doesn't restart each time the megafauna leaves screen...

**Comment by monster860** [comment]
The distance to stop playing music is larger than the distance to stop playing, and there is a fadeout, so if it goes back into range during the fadeout it will fade back in instead of restarting

**Comment by Xoxeyos** [comment]
It'd be cool to have an intro roar for Bubblegum, and the ash drake. I can hear the same *splotch* sound effect from the heirophant and the bubblegum soundtrack. And lastly Bubblegum, Colossus, and the Ash Drake all sound similar. Legion sounds great though.
I think it'd be interesting for each fauna to have their own characteristic *roar*, like Godzilla or something similar.

**Comment by Xoxeyos** [comment]
Here are two things you can use for whatever reason you like, just tossing these in here. 

[Two Sound Effects.zip](https://github.com/yogstation13/Yogstation/files/6013787/Two.Sound.Effects.zip)

I'd upload them as .ogg, but github won't allow it for some reason. ( I made these! )

**Comment by Marmio64** [comment]
did you use beepbox for them?

**Comment by monster860** [comment]
the heck is that

**Review by JamieD1 — APPROVED** [review]

**Comment by Cartlord** [comment]
I know this was a while ago, but I feel like noting that I get hit with a nasty lag spike whenever I approach a megafauna (even as Ghost, so I doubt it's something to do with pathfinding), and I think the music's doing it.
